### ISSUE TITLE
Installation issues Command Prompt Fix Failed _ Revit 2021 Win 10

### ISSUE THREAD

**Issue by carlos-jarch** [eirannejad/pyRevit#1120]
**Describe the bug**
pyRevit is intalled but dos not load.

I Have tried the Fix on the command prompt but it still wont work

```
C:\Users\cvazquez>pyrevit env
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
==> Attachments
==> Installed Extensions
==> Default Extension Search Path
C:\Users\cvazquez\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
2021 First Customer Ship | Version: 21.0.0.383 | Build: 20200220_1100(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2021\"
==> Running Revit Instances
PID: 36284 | 2021 First Customer Ship | Version: 21.0.0.383 | Build: 20200220_1100(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2021"
==> User Environment
Microsoft Windows 10 [Version 10.0.19042]
Executing User: HCS\cvazquez
Active User: HCS\cvazquez
Admin Access: No
%APPDATA%: "C:\Users\cvazquez\AppData\Roaming"
Latest Installed .Net Framework: 4.8
No .Net Target Packs are installed.
No .Ne-Core Target Packs are installed.
pyRevit CLI 4.8.4.0
```

```
C:\Users\cvazquez>pyrevit attach master 277 --installed
Error: Can not find clone "master"
Run with "--debug" option to see debug messages
```

Thank you for your time I am running Revit 2021

**Comment by eirannejad** [comment]
See this comment please https://github.com/eirannejad/pyRevit/issues/1075#issuecomment-734950066

**Comment by carlos-jarch** [comment]
The folder "C:\Users\cvazquez\AppData\Roaming\pyRevit" is empty and the file pyRevit_config is an empty text file

**Comment by carlos-jarch** [comment]
Solved it i downloaded the  CLI installer

**Comment by eirannejad** [comment]
Uh Great. I'm gonna add a note to the CLI download page to make it clear. A couple of users had accidentally downloaded the CLI as well

**Comment by JorgenBoe** [comment]
![Bios pyrevit](https://user-images.githubusercontent.com/78442512/106905437-94d96900-66fc-11eb-8eb7-fc13f939c935.PNG)
Hi 
I can't get revit to run pyrevit.
I got so far, but dont know what do do from here on. Can u guide me what to do now.
Thx

**Comment by eirannejad** [comment]
Would you mind running `pyrevit env` and sharing the results?

**Comment by JorgenBoe** [comment]
Hi Ehsan

My colleague ran it with out a problem – same spec on pc

Fra: Ehsan Iran-Nejad <notifications@github.com>
Sendt: 4. februar 2021 18:14
Til: eirannejad/pyRevit <pyRevit@noreply.github.com>
Cc: Jørgen Lykkegaard Boe <jb@m4arkitekter.dk>; Comment <comment@noreply.github.com>
Emne: Re: [eirannejad/pyRevit] Installation issues Command Prompt Fix Failed _ Revit 2021 Win 10 (#1120)


Would you mind running pyrevit env and sharing the results?
